ledgerManager.php

```diff
@@ -375,8 +375,8 @@ public function retrieveGroupedSumsOfTransactionsAsObjects($startDate, $endDate
 			$endDate = date('Y-m-d', strtotime("now"));
 		}
 		$transactionGroup = new stdClass();
-		$transactionGroup->startDate = $startDate;
-		$transactionGroup->endDate = $endDate;
+		$transactionGroup->startDate = date('M j, Y', strtotime($startDate));
```

Comment:
at some point, i'd like to go through this class and explicitly name returned formatted properties.  ie. ```$transactionGroup->formattedStartDate = ...```